libs/design-system/src/atoms/inputs/inputs.stories.tsx

```diff
@@ -32,54 +26,24 @@ export const SelectPrimary: StoryFn = () => {
 
   return (
     <SelectInput
-      type="primary"
       id="primary"
-      handleChange={(event) => console.log(event.target.value)}
+      handleChange={(value) => console.log(value)}
       options={options}
-      className="plasmo-w-80"
-      required
-    />
-  );
-};
-
-export const SelectSecondary: StoryFn = () => {
-  const options = [
-    { id: '1', label: 'Option 1', value: 'Option 1' },
-    { id: '2', label: 'Option 2', value: 'Option 2' },
-    { id: '3', label: 'Option 3', value: 'Option 3' },
-  ];
-
-  return (
-    <SelectInput
-      type="secondary"
-      id="secondary"
-      handleChange={(event) => console.log(event.target.value)}
-      options={options}
-      className="plasmo-w-40"
-      required
+      value={options[0].value}
+      className="plasmo-w-56"
     />
   );
 };
 
 export const TextArea: StoryFn = () => (
-  <TextAreaInput
-    inputId="comment"
-    handleChange={(event) => console.log(event.target.value)}
-    className="plasmo-w-80"
-    required
-  />
+  <TextAreaInput inputId="comment" handleChange={(event) => console.log(event.target.value)} required />
 );
 
-export const Toggle: StoryFn = () => {
-  const [checked, setChecked] = useState<boolean>(false);
-  const changeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setChecked(event.target.checked);
-    console.log(event.target.checked);
-  };
+export const NewToggle: StoryFn = () => {
+  const changeHandler = (checkedStatus: boolean) => console.log(checkedStatus);
   return (
     <ToggleInput
       options={{ checked: 'Notion', unchecked: 'LinkedIn' }}
-      checked={checked}
       inputId="linkedinNotionSwitch"
       handleChange={changeHandler}
     />
```

Comment:
```suggestion
    <div className="plasmo-w-60">
      <ToggleInput
        options={{ checked: 'Notion', unchecked: 'LinkedIn' }}
        inputId="linkedinNotionSwitch"
        handleChange={changeHandler}
      />
    </div>
```

Just to see that when the size is set the buttons are the right size